whois-nrtm4/src/main/java/net/ripe/db/nrtm4/SnapshotRecordProcessor.java

```diff
@@ -52,6 +51,8 @@ public Map<CIString, byte[]> get() {
             LOGGER.error("Exception writing snapshot {}", e);
             Thread.currentThread().interrupt();
             throw new RuntimeException(e);
+        } finally {
+            resources.values().forEach(GzipOutStreamWriter::close);
```

Comment:
If there's more than one `resources` then do a try/catch here *also*

Reply:
Not needed, GzipOutStreamWriter cvlose method has try catch block

Reply:
Thanks